examples/demo-app/src/cloud-providers/aws/aws-provider.js

```diff
@@ -0,0 +1,292 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import AwsIcon from './aws-icon';
+import {Provider} from 'kepler.gl/cloud-providers';
+import window from 'global';
+import {AWS_WEB_CLIENT_ID, awsmobile} from './aws-exports';
+import 'aws-sdk';
+import {Auth, Storage} from 'aws-amplify';
+import {MAP_URI} from '../../constants/default-settings';
+import {AWS_LOGIN_URL} from './aws-login';
+
+export const PROVIDER_NAME = 'aws';
+export const DISPLAY_NAME = 'AWS';
+export const PRIVATE_STORAGE_ENABLED = true;
+export const SHARING_ENABLED = true;
+
+Storage.configure({level: 'private'});
+
+export default class AwsProvider extends Provider {
+  constructor() {
+    super({name: PROVIDER_NAME, displayName: DISPLAY_NAME, icon: AwsIcon});
+  }
+
+  /**
+   * Required!
+   * Idea / Hack: https://github.com/amazon-archives/amazon-cognito-identity-js/issues/508
```

Comment:
what does it mean exactly in this context?

Reply:
old comment. can be removed.

Reply:
Done